.github/workflows/ci_playground.yml

```diff
@@ -76,7 +76,7 @@ jobs:
   test_cache:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v3
```

Comment:
I think we should put the fix also here, since this script is for testing stuff and later the tested stuff might not work because of the missing history and tags